src/cmd-build

```diff
@@ -119,12 +116,19 @@ if ! git diff --quiet --exit-code; then
 fi
 popd
 commitmeta_input_json=$(pwd)/tmp/commit-metadata-input.json
-cat >"${commitmeta_input_json}" <<EOF
+cat > "${commitmeta_input_json}" <<EOF
 {
   "coreos-assembler.config-gitrev": "${config_gitrev}",
-  "coreos-assembler.config-dirty": ${config_dirty}
+  "coreos-assembler.config-dirty": "${config_dirty}"
 }
 EOF
+
+# Generate an archive metadata about the config GIT checkout, but
+# only if the build is from a Git Recipe.
+if [ -e "${workdir}/src/config-git" ]; then
```

Comment:
this is still a problem for fedora as [mentioned earlier](https://github.com/coreos/coreos-assembler/pull/299#discussion_r252428047)

Maybe something like this would work:

```                                         
if [ -e "${workdir}/src/config-git" ]; then                                         
    config_git_dir="${workdir}/src/config-git"                                      
elif [ -e "${workdir}/src/config" ]; then                                           
    config_git_dir="${workdir}/src/config"                                          
else                                                                                
    fatal "Git config directory doesn't exist"                                      
fi                                                                                  
                                                                                    
prepare_git_artifacts $config_git_dir "$(pwd)/config.tar.gz" "$(pwd)/config-git.json

```

Reply:
Awesome suggestion, its been included and tested.